integration-tests/invoke_test.go

```diff
@@ -76,7 +76,7 @@ func testAzureInvokeVM(t *testing.T, tmpConfigFile string, workingDirectory stri
 
 	stackName := fmt.Sprintf("az-invoke-vm-%s", os.Getenv("CI_PIPELINE_ID"))
 	t.Log("creating vm")
-	createCmd := exec.Command("invoke", "az.create-vm", "--no-interactive", "--stack-name", stackName, "--config-path", tmpConfigFile, "--account", "agent-qa")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
It doesn't work with `agent-qa`? Or we are setting it at setup?

Reply:
I moved it to the config file: https://github.com/DataDog/test-infra-definitions/pull/967/files#diff-5b0d7f0d4cd5bb98edb0f1e6b5ac0747cae4be958137b00d9785e4863a90acdcR12 sorry it was not super explicit. We already do the same thing with AWS so I just centralised everything